### PR TITLE
Park designations

### PR DESCRIPTION
**Pull request by schoure98**
I have create Polar Area Chart for park designations. This component is based on the parks endpoint. I have included the designation witch number of parks at least greater than 8. I selected the number 8 considering it as the average number of park designation criteria. 

Dependency: chart.js

Attaching the expected screenshot below. 

Please let me know your feedback. 

Screenshot 2023-05-26 at 9 42 34 PM

### CODE REVIEW

File: park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+const DesignationPolarArea = (parks) => {
+  const parkData = LoadParkData(parks);
+
+  //Fetch all the park designations
+  const parkDesignations = parkData.map((park) => park.designation);
+
+  //Unique park designations
+  const uniqueDesignations = [...new Set(parkDesignations)];
+  console.log(uniqueDesignations);
```
Comment (juhiaugustine) [inline park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx]: Can you try removing the console logs!

File: park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  //Count of each unique designation
+  const counts = parkDesignations.reduce((acc, designation) => {
+    acc[designation] = (acc[designation] || 0) + 1;
+    return acc;
+  }, {});
+
+  const countsOfDesignations = JSON.stringify(counts);
+  console.log(countsOfDesignations);
```
Comment (juhiaugustine) [inline park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx]: Same as above

File: park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+  const filteredDesignations = Object.entries(counts)
+    .filter(([designation, count]) => count > 8 && designation !== "")
```
Comment (juhiaugustine) [inline park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx:32]: Nice use of .reduce(), and .filter()

**Comment by juhiaugustine** [comment]
Overall it looks good. Just added some minor comments.

**Review by juhiaugustine — APPROVED** [review]

**Review by rutujapadgilwar — COMMENTED** [review]
@schoure98, great job on the polar chart! The unique color scheme and overall appearance are impressive. Please ensure to integrate the recommended changes provided by @juhiaugustine into the codebase

File: park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx (hunk 1 above)
Comment (schoure98) [reply park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx]: Removed

File: park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx (hunk 2 above)
Comment (schoure98) [reply park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx]: Thanks for noticing this, I have removed the console.log statements.

File: park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx (hunk 3 above)
Comment (schoure98) [reply park-dashboard/src/Components/ParkDesignations/DesignationPolarArea.jsx:32]: Thank you.

**Comment by schoure98** [comment]
Thanks @rutujapadgilwar. I have removed the console.log statements.

**Pull request merged**